partcad-cli/src/partcad_cli/click/command.py

```
@@ -180,7 +316,7 @@ def cli(ctx, verbose, quiet, no_ansi, package, format):
 
 
 @cli.result_callback()
-def process_result(result, verbose, quiet, no_ansi, package, format):
+def process_result(result, verbose, quiet, no_ansi, package, format, threads_max, cache_files, cache_files_max_entry_size, cache_files_min_entry_size, cache_memory_max_entry_size, cache_memory_double_cache_max_entry_size, cache_dependencies_ignore, python_sandbox, internal_state_dir, force_update, google_api_key, openai_api_key, ollama_num_thread, max_geometric_modeling, max_model_generation, max_script_correction):
```

Ruff (0.8.2)

Missing return type annotation for public function `process_result`

Add return type annotation: `None`

(ANN201)